### ISSUE TITLE
On KIE Sandbox it's not possible to use the import DMN on an Business Rule node after renaming the file

### ISSUE THREAD

**Issue by ljmotta** [apache/incubator-kie-issues#530]
The KIE Sandbox FS hangs if you try to import a DMN in a BPMN that was renamed after the node was created. All the following edits will not be saved, but If you delete the Business Rule node, the FS will be recovered.

Steps to reproduce:
- Create a new BPMN;
- Create a new activitity node;
- Convert the node to a Business Rule;
- Rename the file;
- Try to import a DMN;

## video
https://github.com/kiegroup/kie-issues/assets/24302289/e5aac8fb-e1bc-47b4-b681-bfdfd1631e03

```[tasklist]
### Tasks
- [ ] It should be possible to import an DMN after the file is renamed.
```

**Comment by inodeman** [comment]
Hi @tiagobento checking on it. Will let you know an ETA

**Comment by eduardocerqueira** [comment]
for planning purpose, @inodeman can you pls share an update on this? how close are you to send a PR and when could we expect for it?

**Comment by inodeman** [comment]
As you are well aware @eduardocerqueira been in medical leave for the last weeks so I have not looked at it closely, also if I recall correctly you told me not to focus on GWT issues that you had reassigned it. Let me know if that changed

**Comment by eduardocerqueira** [comment]
nothing changed, this issue was brought up to my attention during a meeting today and I was asking for an update as I noticed it was still assigned to you. Just unassigned it, we will add it back to the plan in the next planning cycle.

**Comment by jomarko** [comment]
I was not able to reproduce this issue as of 16of July 2024 using https://sandbox.kie.org/dev. Not sure if there is a step I do differently than the reporter did.

**Comment by ljmotta** [comment]
@jomarko I still get the error. Did you renamed the file? This step is crucial.

**Comment by jomarko** [comment]
I renamed the BPMN file, should I rename DMN file? I think that is not clear from the ticket description.

**Comment by ljmotta** [comment]
No, just the BPMN file. But you must rename the file after creating the node.

**Comment by inodeman** [comment]
This should do it On 29 Jul 2024, at 8:00 a.m., Luiz João Motta ***@***.***> wrote:﻿
No, just the BPMN file. But you must rename the file after creating the node.

**Comment by tiagobento** [comment]
Fixed in https://github.com/apache/incubator-kie-tools/pull/3421
